tests/run.sh

```diff
@@ -66,13 +65,12 @@ export AWS_ACCESS_KEY_ID=$(echo  "${RES}" | jq --raw-output '.Credentials.Access
 export AWS_SESSION_TOKEN=$(echo "${RES}" | jq --raw-output '.Credentials.SessionToken')
 
 ### HANDLE SSH KEY ###
-echo -e "\\e[36m Uploading SSH key-pair to AWS...\\e[0m"
+echo -e "\\e[36m Generating SSH key-pair...\\e[0m"
 if [ ! -f "$HOME/.ssh/id_rsa.pub" ]; then
   #shellcheck disable=SC2034
   SSH=$(ssh-keygen -b 2048 -t rsa -f "${HOME}/.ssh/id_rsa" -N "" < /dev/zero)
 fi
-aws ec2 import-key-pair --key-name "${CLUSTER_NAME}" --public-key-material "file://$HOME/.ssh/id_rsa.pub"
-export TF_VAR_tectonic_aws_ssh_key="${CLUSTER_NAME}"
+export TF_VAR_tectonic_admin_ssh_key="$(< "$HOME/.ssh/id_rsa.pub")"
```

Comment:
[`$(< ...)` is a Bashism][1].  For POSIX compat, can we use `$(cat ~/.ssh/id_rsa.pub)` (which also replaces `$HOME` with [POSIX-compatible `~` expansion][2])?

[1]: https://mywiki.wooledge.org/CommandSubstitution
[2]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_01